location/detector_test.go

```diff
@@ -16,6 +16,7 @@ func TestDetectorDetectCountry(t *testing.T) {
 		{"95.85.39.36", "NL", ""},
 		{"127.0.0.1", "", ""},
 		{"8.8.8.8.8", "", "failed to parse IP"},
+		{"185.243.112.225", "", ""},
```

Comment:
If we return error when we are unable to found country in database, using country detector would be much easier - if error was not returned, that means country was returned :)

Reply:
return errors.New("unkown country for IP: x.x.x")